stdlib/public/core/UnicodeScalar.swift

```diff
@@ -258,6 +258,16 @@ extension UnicodeScalar : CustomStringConvertible, CustomDebugStringConvertible
   }
 }
 
+extension UnicodeScalar : LosslessStringConvertible {
```

Comment:
I see.  It was not obvious which API it tests, could you add the string `UnicodeScalar.init?(_: String)` in the test name somewhere?  We usually name the tests after specific APIs, if that's what we are targeting.